po/vi.po

```diff
@@ -3389,29 +3391,42 @@ msgid ""
 "keyword can be used for early return, but the \"bare value\" form is "
 "idiomatic at the end of a function (refactor `gcd` to use a `return`)."
 msgstr ""
+"Biểu thức cuối cùng trong hàm (hay một khối) sẽ trở thành giá trị tả về. Đơn "
+"giản bằng cách bỏ qua dấu `;` ở cuối biểu thức. Từ khóa `return` có thể sử "
+"dụng để trả về sớm, nhưng \"giá trị trần (bare value)\" là triết lí ngôn ngữ "
+"(idiomatic) khi đặt ở cuối hàm (tái cấu trúc `gcd` dể sử dụng `return`)."
 
 #: src/control-flow-basics/functions.md
 msgid ""
 "Some functions have no return value, and return the 'unit type', `()`. The "
 "compiler will infer this if the `-> ()` return type is omitted."
 msgstr ""
+"Một số hàm sẽ không có giá trị trở về, sẽ trả về 'kiểu dữ liệu', `()`. Trình "
+"biên dịch sẽ kết luận rằng kiểu dữ liệu sẽ là `-> ()` nếu kiểu dữ liệu trả "
+"về được bỏ qua."
 
 #: src/control-flow-basics/functions.md
 msgid ""
 "Overloading is not supported -- each function has a single implementation."
 msgstr ""
+"Overloading (phương thức nập chống) không được hỗ trợ -- mỗi hàm chỉ có duy "
```

Comment:
The translated term (phương thức nập chống) seems a little unusual. We might just keep the original word.